WireDomain/Sources/WireDomain/Repositories/ConversationsLabels/ConversationLabelsRepository.swift

```diff
@@ -20,19 +20,32 @@ import CoreData
 import WireAPI
 import WireDataModel
 
-/// Facilitate access to conversation labels related domain objects.
+// sourcery: AutoMockable
+/// Facilitates access to conversation labels related domain objects.
+public protocol ConversationLabelsRepositoryProtocol {
 
-protocol ConversationLabelsRepositoryProtocol {
-
-    /// Pull conversation labels from the server and store locally
+    /// Pulls conversation labels from the server and stores locally
 
     func pullConversationLabels() async throws
+
+    /// Updates conversation labels locally
+    /// - parameters:
+    ///     - conversationLabels: The conversation labels to update locally.
+
+    func updateConversationLabels(
+        _ conversationLabels: [ConversationLabel]
+    ) async throws
 }
 
-final class ConversationLabelsRepository: ConversationLabelsRepositoryProtocol {
+public class ConversationLabelsRepository: ConversationLabelsRepositoryProtocol {
+
+    // MARK: - Properties
 
     private let userPropertiesAPI: any UserPropertiesAPI
     private let context: NSManagedObjectContext
+    private let logger = WireLogger(tag: "conversation-labels")
```

Comment:
I guess we manually set the tag because WireLogger is not on WireFoundation, right?

Reply:
WireLogger is not part of WireFoundation, it is part of WireSystem. 
The tag didn't exist before, I created it.